### PR TITLE
adding ssh_key is now possible for cloning sub

### PR DESCRIPTION
**Pull request by Albanetonnellier**
adding ssh_key is now possible for cloning submodules

### CODE REVIEW

File: website/docs/using-qovery/configuration/application.md [hunk 1]
```diff
@@ -273,6 +275,23 @@ To learn how to set up environment variables in your projects and applications,
 
 To learn how to set up secrets in your projects and applications, navigate to [configuring Secrets][docs.using-qovery.configuration.secret] section.
 
+If you have a repository with private submodules.
+We can get them if you give us an ssh key.
+
+To do so, just add:
+
+  ```bash
```
Comment (evoxmusic) [inline website/docs/using-qovery/configuration/application.md:283]: @Albanetonnellier - can you remove the space/tab?

File: website/docs/using-qovery/configuration/application.md [hunk 2]
```diff
@@ -273,6 +275,23 @@ To learn how to set up environment variables in your projects and applications,
...
+  GIT_SSH_KEY_xxx
```
Comment (evoxmusic) [inline website/docs/using-qovery/configuration/application.md:284]: same here

Comment (evoxmusic) [inline website/docs/using-qovery/configuration/application.md:284]: @Albanetonnellier can you explain what is the `_xxx`. What do you expect? Can we have multiple SSH private keys and passphrases? How does it work?

File: website/docs/using-qovery/configuration/application.md.erb [hunk 3]
```diff
@@ -261,6 +263,23 @@ To learn how to set up environment variables in your projects and applications,
 
 To learn how to set up secrets in your projects and applications, navigate to [configuring Secrets][docs.using-qovery.configuration.secret] section.
 
+If you have a repository with private submodules.
+We can get them if you give us an ssh key.
+
+To do so, just add:
+
+  ```bash
```
Comment (evoxmusic) [inline website/docs/using-qovery/configuration/application.md.erb:271]: same here

File: website/docs/using-qovery/configuration/application.md.erb [hunk 4]
```diff
@@ -261,6 +263,23 @@ To learn how to set up environment variables in your projects and applications,
...
+  GIT_SSH_KEY_xxx
```
Comment (evoxmusic) [inline website/docs/using-qovery/configuration/application.md.erb:272]: same here

**Review by evoxmusic — CHANGES_REQUESTED** [review]

File: website/docs/using-qovery/configuration/application.md (hunk 2 above)
Comment (Albanetonnellier) [reply website/docs/using-qovery/configuration/application.md:284]: the _xxx is to tell that you can put whatever name you want at the end for example GIT_SSH_KEY_PRODUCTION
we will take the ssh keys in the env starting with GIT_SSH_KEY
and we can have multiples ssh private keys.

**Comment by alecarrano** [comment]
closing since submodules are already documented

**Pull request closed without merging**